.editorconfig

```diff
@@ -0,0 +1,6 @@
+# Editor configuration, see https://editorconfig.org
+[*]
+charset = utf-8
```

Comment:
@gnbm was this intented?

Reply:
It was. To make sure the editor configs work equally between Mac and Windows..hopefully
Did you find any impact on this?

Reply:
No, just curious 👍